src/metatensor/models/utils/normalize.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Union
+
+import torch
+from metatensor.learn.data.dataset import _BaseDataset
+from metatensor.torch import TensorBlock, TensorMap
+
+
+def get_average_number_of_atoms(
```

Comment:
all the new code in `utils` should be added to the API reference in the developers documentation!

Reply:
Outdated and removed after #83

Reply:
Done